File: 프로그래머스/unrated/181939. 더 크게 합치기/더 크게 합치기.java
class Solution {
    public int solution(int a, int b) {
        String a_b = String.valueOf(a) + String.valueOf(b);
        String b_a = String.valueOf(b) + String.valueOf(a);
        if(Integer.valueOf(a_b) > Integer.valueOf(b_a)){
            return Integer.valueOf(a_b);
        }
            
        return Integer.valueOf(b_a);
    }
}

File: 프로그래머스/unrated/181939. 더 크게 합치기/더 크게 합치기.cpp
#include <string>
#include <vector>
#include <iostream>
using namespace std;

int solution(int a, int b) {
    int answer = 0;
    string a_b = to_string(a) + to_string(b);
    string b_a = to_string(b) + to_string(a);
    if(stoi(a_b) > stoi(b_a)) return stoi(a_b);
    return stoi(b_a);
}